test/pkcs12_format_test.c

```diff
@@ -211,6 +216,52 @@ static const PKCS12_ENC mac_default = {
     1000
 };
 
+static const int enc_nids[] = {
+    /* NOTE: To use PBES2 we pass the desired cipher NID instead of NID_pbes2 */
+    NID_aes_128_cbc,
+    NID_aes_256_cbc,
+    NID_des_ede3_cbc,
+/*
+ * TODO: Many algorithms are defined but are not working. Determine if they
```

Comment:
Drop the TODO, just note that these aren't working here.
Maybe raise an issue describing this so it isn't lost.